packages/x-components/cypress.config.ts

```diff
@@ -9,34 +7,41 @@ export default defineConfig({
     requestTimeout: 7000,
     viewportHeight: 1080,
     viewportWidth: 1920,
-    screenshotOnRunFailure: false,
-    video: false,
+    specPattern: 'tests/e2e/**/*.feature',
     supportFile: 'tests/support/index.ts',
     fixturesFolder: 'tests/e2e/fixtures',
     screenshotsFolder: 'tests/e2e/screenshots',
+    experimentalRunAllSpecs: true,
+    screenshotOnRunFailure: false,
+    video: false,
     retries: {
       openMode: 0,
       runMode: 1
     },
     async setupNodeEvents(on, config) {
-      await addCucumberPreprocessorPlugin(on, config);
-      on(
-        'file:preprocessor',
-        require('@bahmutov/cypress-esbuild-preprocessor')({
-          plugins: [createEsbuildPlugin(config)]
-        })
+      const { createEsbuildPlugin } = require('@badeball/cypress-cucumber-preprocessor/esbuild');
+      const createBundler = require('@bahmutov/cypress-esbuild-preprocessor');
+
+      // await here
```

Comment:
Maybe we should improve this comment.

Reply:
Yep. I rewrote this config following the official doc for cucumber